packages/core/src/components/drawer/drawer.tsx

```diff
@@ -130,10 +131,10 @@ export class Drawer extends AbstractPureComponent<DrawerProps> {
                       [isPositionHorizontal(realPosition) ? "height" : "width"]: size,
                   };
         return (
-            <Overlay {...this.props} className={Classes.OVERLAY_CONTAINER}>
+            <Overlay {...overlayProps} className={classNames({ [Classes.OVERLAY_CONTAINER]: hasBackdrop })}>
```

Comment:
I realize now that this "overlay container" class (added in https://github.com/palantir/blueprint/pull/2957) has a somewhat misleading name since it's only used by the Drawer component and it's non-descriptive about its purpose... it might make more sense to call this "OVERLAY_FIXED_BACKDROP" or something like that. We can't change it now though; it's part of the public API... just musing about the name here, perhaps I'll add some code comments about this in a separate PR.